docs/docsite/rst/community/steering/steering_committee_membership.rst

```diff
@@ -89,7 +89,7 @@ In case of absence or irregular participation, the involuntarily removal process
 #. Another Committee member (hereinafter the initiator) contacts the person by email asking if they are still interested in fulfilling their Committee member's duties.
 #. If they respond that they are not interested, the initiator asks the person to step down on their own following the :ref:`Voluntarily leaving process<Voluntarily leaving process>`.
 #. If there has been no response or stepping down issue created by the person within a reasonable time, the initiator notifies other Committee members about the situation.
-#. In case of agreement among the Committee about the need for removal, the initiator provides a draft of a corresponding topic's description to the Committee via email for discussion and approval.
+#. In case of agreement among the Committee about the need for removal, the initiator provides a draft of a corresponding topic's description to the Committee with email for discussion and approval.
```

Comment:
```suggestion
#. In case of agreement among the Committee about the need for removal, the initiator provides a draft of a corresponding topic's description to the Committee by email for discussion and approval.
```

Reply:
Updated via commit [37845f2](https://github.com/ansible/ansible-documentation/pull/479/commits/37845f28f96cb2c8ba39e95aaae41aa60332d298)